### PR TITLE
GPIB-66 add pie chart

### PR DESCRIPTION
**Pull request by callumjg**

### CODE REVIEW

File: src/components/AddressPie.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { RadialChart } from "react-vis";
+const AddressPie = ({ addresses = [], ...props }) => {
+  const data = addresses.slice(0, 2).map((a) => ({
```
Comment (bitcoinbrisbane) [inline src/components/AddressPie.js:4]: Whats the (0,2)?

Comment (callumjg) [reply src/components/AddressPie.js:4]: Returns a new array with only the first two items

Comment (callumjg) [reply src/components/AddressPie.js:4]: This is because users will only ahve 2 addresses, but the test user we have has about 8 or 9

**Comment by sjustintaylor** [comment]
I'm assuming that this is what the user story is supposed to look like
Screenshot from 2020-06-11 15-23-23

Other than that, everything works as expected

**Comment by callumjg** [comment]
> I'm assuming that this is what the user story is supposed to look like
> Screenshot from 2020-06-11 15-23-23
> 
> Other than that, everything works as expected

Haha, yep, definitely not very pretty at this stage

**Review by sjustintaylor — APPROVED** [review]

File: src/components/AddressPie.js (hunk 1 above)
Comment (bitcoinbrisbane) [reply src/components/AddressPie.js:4]: Cant we make it for any number of addresses?

**Comment by bitcoinbrisbane** [comment]
Cool, but we need to extend to have more than 2.  Maybe another story.

**Pull request merged**